<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigsBackgrounds</class>
 <widget class="QWidget" name="ConfigsBackgrounds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxVisual">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
     <property name="title">
      <string>Visualização</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Lista de fundos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" rowspan="2">
       <widget class="QListWidget" name="listWidgetBgs">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonAddBg">
        <property name="text">
         <string>Adicionar fundo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxBg">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Camadas do fundo</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Camadas:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QListWidget" name="listWidgetLayers">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonAddLayer">
        <property name="text">
         <string>Adiciona camada</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButtonRemoveLayer">
        <property name="text">
         <string>Remove camada</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxLayer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Configurações da camada</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Imagem:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="7">
       <widget class="QLineEdit" name="lineEditImageFile">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2" colspan="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Deslocamento:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBoxDx">
        <property name="minimum">
         <number>-9999999</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QSpinBox" name="spinBoxDy">
        <property name="minimum">
         <number>-999999999</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Repete:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="3">
       <widget class="QCheckBox" name="checkBoxRepeatHor">
        <property name="text">
         <string>Horizontalmente</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5" colspan="3">
       <widget class="QCheckBox" name="checkBoxRepeatVer">
        <property name="text">
         <string>Verticalmente</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Comportamento:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="8">
       <widget class="QComboBox" name="comboBoxBehavior">
        <item>
         <property name="text">
          <string>Estatica</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Flutuando</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Flutuando com relação a posição do personagem</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Flutuando com velocidade</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Velocidade:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QSpinBox" name="spinBoxVx">
        <property name="minimum">
         <number>-9999</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="6" colspan="2">
       <widget class="QSpinBox" name="spinBoxVy">
        <property name="minimum">
         <number>-99999</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
